### PR TITLE
Added a Policy Card Reminder Dialogue Box 

### PR DESCRIPTION
**Pull request by sparrow8332**
Adds in a Dialogue Box to remind player to change policy cards - continue without changes or auto opens government screen if changes are wanted to be made

### CODE REVIEW

**Comment by Azurency** [comment]
I don't know how I feel about a popup that show itself when you click on the action button. At the very least there should be a setting to turn it on/off.

I think the idea was to rework the notification system and then add notification for this kind of things and not only a status message that is easily missed. I think the tech/civic popup + a status message + a notification should be sufficient to remember to change the policies.

I will give your code a try later today, would be nice if we could have more feedback on this.

**Comment by deggesim** [comment]
I'd find this kind of dialogue box very useful.
Sometimes I forget to change my policies (or at least check if I really want to change) and have to reload.

**Comment by sparrow8332** [comment]
maybe have it default off but option to turn on in the settings so each user can choose if they would need it or not. The reason i was going for the dialogue box more than a notification banner like tech/civic was because even they are easily  missed lol

**Comment by sparrow8332** [comment]
Added Setting for Policy Reminder

File: Assets/UI/actionpanel.lua [hunk 1]
```diff
@@ -553,6 +559,19 @@ function DoEndTurn( optionalNewBlocker:number )
...
   end
 
+  function CQUI_HidePRD()
+	if CQUI_ShowPRD then
+	  LuaEvents.CQUI_clearUnitPath();
+	end
+  end
+  
+  function CQUI_OnSettingsUpdate()
+	CQUI_ShowPRD = GameConfiguration.GetValue("CQUI_ShowPolicyReminder");
```
Comment (Azurency) [inline Assets/UI/actionpanel.lua]: (small detail) indentation missing 😉

File: Assets/cqui_settings.sql [hunk 2]
```diff
@@ -41,7 +41,8 @@ INSERT OR REPLACE INTO CQUI_Settings -- Don't touch this line!
       ("CQUI_BlockOnCityAttack", 1), -- Block turn from ending if you have a city that can attack
       ("CQUI_ProductionQueue", 1), -- A production queue appears next to the production panel, allowing multiple constructions to be queued at once
       ("CQUI_ShowCultureGrowth", 1), -- Shows cultural growth overlay in cityview
-      ("CQUI_ShowLuxuries", 1), -- Luxury resources will show in the top-bar next to strategic resources
+      ("CQUI_ShowPolicyReminder", 0), 
+	  ("CQUI_ShowLuxuries", 1), -- Luxury resources will show in the top-bar next to strategic resources
```
Comment (Azurency) [inline Assets/cqui_settings.sql]: (small detail) why is there a tab in this indentation ?

**Comment by Azurency** [comment]
I think it's still a little "raw" maybe we could remind the cards that changed in the popup (like the civic completed popup in fact) and we should also visually add an actual action button and not just the "next turn" (maybe the actual `ICON_NOTIFICATION_CHOOSE_CIVIC` it seems appropriate)

**Comment by sparrow8332** [comment]
untitled

**Pull request closed without merging**